cmd/amp/cli/logs.go

```diff
@@ -0,0 +1,74 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/appcelerator/amp/api/client"
+	"github.com/appcelerator/amp/api/rpc/logs"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"log"
+	"strconv"
+)
+
+// Logs fetches the logs
+func Logs(amp *client.AMP, cmd *cobra.Command) error {
+	ctx, err := amp.GetAuthorizedContext()
+	if err != nil {
+		return err
+	}
+	if amp.Verbose() {
+		fmt.Println("Logs")
+		fmt.Printf("service_id: %v\n", cmd.Flag("service_id").Value)
+		fmt.Printf("service_name: %v\n", cmd.Flag("service_name").Value)
+		fmt.Printf("message: %v\n", cmd.Flag("message").Value)
+		fmt.Printf("container_id: %v\n", cmd.Flag("container_id").Value)
+		fmt.Printf("node_id: %v\n", cmd.Flag("node_id").Value)
+		fmt.Printf("from: %v\n", cmd.Flag("from").Value)
+		fmt.Printf("n: %v\n", cmd.Flag("n").Value)
+		fmt.Printf("short: %v\n", cmd.Flag("short").Value)
+	}
+	conn, err := grpc.Dial(client.ServerAddress, grpc.WithInsecure())
```

Comment:
This belongs in `api/client/amp.go` (or perhaps `api/client/logs.go`). This is the client interface that is independent of the CLI (or web UI, etc). See the next comment for the whole picture.